### ISSUE TITLE
NTR: Quantiferon gold test positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8767]
New term request Quantiferon gold test positive (nadea3_190131011154): Positive TB test

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000799
- label: Positive Quantiferon tuberculosis test
- parents: [Positive bloodstream infectious agent antigen test (HP:0430109)](https://hpo.jax.org/app/browse/term/HP:0430109)
- pmid: PMID:29743310
- def: A positive Quantiferon test result indicating Mycobacterium tuberculosis. The Quantiferon assay is designed to stimulate IFN-gamma release from CD4+ T cells in a single tuberculosis antigen tube using long peptides from three M. tuberculosis antigens.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8767
- synonyms:
